<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFAA69;</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>691</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="picLabel3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>You may also like</string>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel8">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>360</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>121</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="picLabel4">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>120</width>
       <height>160</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"> border-style: inset;
border-color:#0F7C8A;
    border-width: 4px;
    border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New arrivals</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>J. R. R. Tolkien</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>Victor Hugo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>270</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>Alexandre Dumas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>270</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>Leo Tolstoy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="hobbitButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>The Hobbit</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="lesMisButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Les Miserables</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="blackTulipButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>The Black Tulip</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="warButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>240</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>War and Peace</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>550</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>George Orwell</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>Frank Herbert</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>560</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>J. R. R. Tolkien</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>550</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#C3C3C3;
</string>
     </property>
     <property name="text">
      <string>Leo Tolstoy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="lotrButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>520</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>The Lord of the Rings</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="duneButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dune</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="annaButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>520</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Anna Karenina</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="orwellButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>520</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1984</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Your Books</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>User User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLinedit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>251</width>
      <height>43</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-width: 2px;
border-radius: 20px;

padding: 6px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="accountPicLabel11">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
border-style: outset;
 border-width: 2px;
    border-radius: 10px;
border-color: beige;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
